### PR TITLE
[stable/traefik] Add configurable logLevel

### PR DESCRIPTION
**Pull request by paul-leathwood**
Signed-off-by: Paul Leathwood <paul.leathwood@sky.uk>

#### What this PR does / why we need it:

The chart overrides the default `logLevel` (`error`) with `info`, because of this `traefik` accounts for 90% of the the total logs in our kubernetes cluster. This PR adds `logLevel` as a configurable value with a default of `error`

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/traefik/templates/configmap.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ data:
     {{- if .Values.debug.enabled }}
     debug = true
     {{- else }}
-    logLevel = "INFO"
+    logLevel = "{{ .Values.logLevel }}"
```
Comment (dtomcej) [inline stable/traefik/templates/configmap.yaml]: I don't have a problem with this, but can you default this to `INFO` for backwards compatibility?

File: stable/traefik/values.yaml [hunk 2]
```diff
@@ -30,6 +30,8 @@ resources: {}
 debug:
   enabled: false
 
+logLevel: error
```
Comment (dtomcej) [inline stable/traefik/values.yaml]: Can you have this disabled by default (commented out)?

**Review by dtomcej — CHANGES_REQUESTED** [review]

File: stable/traefik/values.yaml (hunk 2 above)
Comment (paul-leathwood) [reply stable/traefik/values.yaml]: Isn't this the default? Would it be better if I set this to `info` as per your previous comment?

Comment (paul-leathwood) [reply stable/traefik/values.yaml]: @dtomcej am I correct here and this is the default or do I need to put an `if` statement in?

Thanks,
Paul

Comment (dtomcej) [reply stable/traefik/values.yaml]: There is actually a default that you can put in the template, which works well for this instance!

https://helm.sh/docs/chart_template_guide/#using-the-default-function

File: stable/traefik/templates/configmap.yaml (hunk 1 above)
Comment (dtomcej) [reply stable/traefik/templates/configmap.yaml]: https://helm.sh/docs/chart_template_guide/#using-the-default-function

default INFO :)

File: stable/traefik/values.yaml (hunk 2 above)
Comment (paul-leathwood) [reply stable/traefik/values.yaml]: ah brilliant, learn something new every day. I have made the requested changes

**Pull request merged**